primeqa/services/factories.py

```diff
@@ -12,6 +12,7 @@
 from primeqa.components.reader.extractive import ExtractiveReader
 
 from primeqa.components.retriever.dense import ColBERTRetriever
+from primeqa.components.retriever.dense import DPRRetriever
```

Comment:
Since both `ColBERTRetriever` and `DPRRetriever` are implemented in `dense.py`. I would recommended importing them in one line

```
from primeqa.components.retriever.dense import ColBERTRetriever, DPRRetriever
```

Reply:
Updated as suggested.